operator/controllers/actionsrunner/util/converters.go

```diff
@@ -30,6 +32,16 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+var (
+	runnerImageName    = EnvVar("KUBEACTIONS_RUNNER_IMAGE_NAME", "inloco/kube-actions")
```

Comment:
I'd rather injecting this info via a parameter on a constructor method... in this case you avoid coupling your package with the input method (environment variables/or environment variables defaulting to build information)

Also a good pkg for defining a `Configuration` object using env vars  - https://github.com/Netflix/go-env